### PR TITLE
Adds More Clown Names

### PR DESCRIPTION
**Pull request by Shadowmech88**
I was surprised at how short the list of possible names was.

### CODE REVIEW

**Comment by Intigracy** [comment]
Perfect.

**Comment by ihadtoregisterforthis** [comment]
You forgot
cpt spaulding

**Comment by Shadowmech88** [comment]
@ihadtoregisterforthis 
Fixed.

**Comment by TheFriggenSawbones** [comment]
wheres my boy, Sweet Tooth aka Needles Kane

otherwise the new additions look fantastic

**Comment by 9600bauds** [comment]
We have successfully turned a reference into an in-joke

**Comment by xpcybic** [comment]
> Aunt Scootaloo

You big bully

**Comment by Intigracy** [comment]
You mean an in-joke into a reference.

**Comment by 9600bauds** [comment]
A reference into an in-joke into a reference of an in-joke that is actually once again a reference

**Comment by TheFriggenSawbones** [comment]
@Shadowmech88 bless you

File: config/names/clown.txt [hunk 1]
```diff
@@ -34,3 +34,16 @@ Giggles
 Jingle
 Candy
 Shiggy Diggintons
+Hingle McCringleberry
+Pagliacci
+Coco
+Blinko
+Shaggy Two Dope
+Aunt Scootaloo
```
Comment (kol1th) [inline config/names/clown.txt:42]: oh fuck hahahahhah

Comment (ghost) [inline config/names/clown.txt:42]: not this shittu again

**Comment by ghost** [comment]
why isn't the biggest clown of all, @9600bauds in this list?

**Comment by 9600bauds** [comment]
https://www.youtube.com/watch?v=WqkjYKUXERQ

**Comment by PJB3005** [comment]
Quick Celt reference

**Comment by Intigracy** [comment]
Celt doesn't deserve a reference though.

**Comment by ghost** [comment]
I agree

File: config/names/clown.txt (hunk 1 above)
Comment (NigglyWiggly) [inline config/names/clown.txt:42]: Heh

**Comment by NigglyWiggly** [comment]
> No 'Pogo the Clown'

pogo_the_clown_-_01

File: config/names/clown.txt (hunk 1 above)
Comment (Probe1) [inline config/names/clown.txt:42]: I can't wait to see what happens in 4 years when someone finds it and removes it.

**Comment by Brolaire** [comment]
Do Jesters get the same name suggestions or do they have seperate ones?

**Comment by clusterfack** [comment]
:cry:

**Comment by Shadowmech88** [comment]
@Brolaire 
Since Jester is just an alternate title for Clown, I believe they use the same set of names.

**Comment by carshalash** [comment]
Merge it for meme magic.

**Comment by Brolaire** [comment]
it happened

**Comment by clusterfack** [comment]
:sob:

**Comment by Probe1** [comment]
:eggplant:

**Pull request merged**